front/src/components/UI/ProductView/ProductFilter.tsx

```diff
@@ -0,0 +1,26 @@
+import { ProductFilterStyle } from "./ProductView.styled";
+
+interface ProductsFilterProps {
+  onChangeFilter: (arg0: string) => void;
+  value: string;
+}
+
+interface HandlerProps {
```

Comment:
niepotrzebnie, są już gotowe typy napisane i TS je dostarcza. To jakiś zwykły Event